store/extensions/Outfit Copy/extension.json

```
@@ -0,0 +1,40 @@
+{
+    "title": "Outfit Copy",
+    "description": "Copy everyone's outfit.\nCopy someone \":copy x\"\nCombine outfits: \":outfitcombine x y\"\nCycle through the outfits of everyone in the room: \":outfitcycle\"\nTo stop: \":outfitstop\n",
+    "authors": [
+      {
+        "name": "altxnor",
+        "hotel": "",
+        "username": "",
+        "discord": "",
+        "twitter": ""
```

Leave these out if you're gonna leave them blank
```suggestion
        "name": "altxnor"
```